CKF.py

```diff
@@ -0,0 +1,166 @@
+import numpy as np
+from scipy.integrate import solve_ivp
+import matplotlib.pyplot as plt
+from abc import ABCMeta, abstractmethod
+from propagator import *
+import ad
+from ad import jacobian
+from FakeMeasurements import genMSM
+
+Re = 6378.1363 #km
+mu = 3.986004415e5 #km^3/s^2
+J2 = 0.0010826269 #unitless
+J3 = 0   #we are not modeling J3
```

Comment:
All of these values are in wms.__init__. We should probably call them from there